src/entities/mellow/mellowLpRouter.ts

```diff
@@ -80,6 +80,8 @@ class MellowLpRouter {
 
   public isRegisteredForAutoRollover = false;
 
+  private canManageVaultPositions?: boolean[];
```

Comment:
does it have to be nullable? you can just do
```suggestion
  private canManageVaultPositions: boolean[] = [];
```

Reply:
I think it helps to keep its length consistent with the length of the ERC20RootVaultAddresses vector.

Reply:
then does it make sense to keep this as property in the class if you can compute it using another property?